storage/innobase/sync/sync0rw.cc

```diff
@@ -654,6 +657,12 @@ rw_lock_x_lock_func(
 	ut_ad(rw_lock_validate(lock));
 	ut_ad(!rw_lock_own(lock, RW_LOCK_S));
 
+	if (rw_lock_x_lock_low(lock, pass, file_name, line)) {
+		/* Locking succeeded */
+		return;
+	}
+	rw_lock_stats.rw_x_spin_wait_count.inc();
```

Comment:
Same concern as above: is it supposed to be `wait_count` or `spin_wait_count`?

Reply:
For each type of lock it has this 3 types:

        int64_counter_t         rw_s_spin_wait_count;
        int64_counter_t         rw_s_spin_round_count;
        int64_counter_t         rw_s_os_wait_count;

spin_round_count would represent round completed and spin_wait_count represents spin-waits.
os_wait_count represents the os-wait which is rare if the spin-cycle fails to obtain the lock.